lib/vmdb/appliance.rb

```diff
@@ -146,24 +146,40 @@ def self.get_network
       retVal
     end
 
-    def self.init_diagnostics
-      log_dir = "/var/www/miq/vmdb/log"
-      gem_log = File.join(log_dir, "gem_list.txt")
-      rpm_log = File.join(log_dir, "package_list_rpm.txt")
+    def self.installed_rpms
+      File.open(File.join(log_dir, "package_list_rpm.txt"), "a") do |file|
+        puts file, "start: date time is: #{Time.now.utc}"
```

Comment:
This will write the file handle to the screen followed by the styff....I think you want `file.puts "stuff"`